sqlsorcery.py

```diff
@@ -35,29 +35,47 @@ def __init__(
         self.engine = self._get_engine()
 
     def _cstr(self):
+        """
+        Constructs SQLAlchemy connection string.
+        """
         return URL(self.dialect, **self.options)
 
-    def _bigquery_options(self):
+    def _add_bigquery_options(self):
+        """
+        Adds BigQuery specific connection options for creating engine.
+        """
         if self.dialect == "bigquery":
-            self._get_creds()
+            self._get_bigquery_creds()
             self.helpers["credentials_path"] = self.creds
 
-    def _get_creds(self):
+    def _get_bigquery_creds(self):
```

Comment:
This is only called once, inside of another private function, and doesn't return any values. Thoughts on just putting the code inside `_add_bigquery_options` instead of a separate function?